### PR TITLE
Correct vagrant boot of boxes

### PR DESCRIPTION
**Pull request by andyp1per**
Our Vagrant file is incorrectly configuring the box to boot in that it is not configuring it at all and we always pick up disco64 as the default. This PR fixes this. It also makes some attempt to make sure that we install available pre-requisites for different ubuntu editions as these vary. I have corrected this for xenial, bionic and disco only.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -82,65 +83,55 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
...
-    config.vm.provision :shell, path: "Tools/vagrant/initvagrant.sh"
-    config.vm.provider "virtualbox" do |vb|
+    artful32.vm.box = "ubuntu/artful32"
+    artful32.vm.provision :shell, path: "Tools/vagrant/initvagrant.sh"
+    artful32.vm.provider "virtualbox" do |vb|
       vb.name = "ArduPilot (artful32)"
     end
   end
 
-  # 18.04 LTS
```
Comment (peterbarker) [inline Vagrantfile]: We prefer to leave these in as people may still have boxes around they use.

Comment (andyp1per) [reply Vagrantfile]: Fixed

**Comment by andyp1per** [comment]
I have verified that the changes to install-prereqs-ubuntu.sh work in a windows WSL environment

**Comment by peterbarker** [comment]
@khancyr I reckon pretty much everything has to have it now.  And Ubuntu definitely will.

If it breaks, i'll fix it if nobody else does :-)

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**